src/operator/optimizer_op-inl.h

```diff
@@ -1088,6 +1090,146 @@ inline void FtrlUpdateEx(const nnvm::NodeAttrs& attrs,
   }
 }
 
+
+// Implementation for signSGD and Signum
+
+struct SignSGDParam : public dmlc::Parameter<SignSGDParam> {
+  float lr;
+  float wd;
+  float rescale_grad;
+  float clip_gradient;
```

Comment:
If the `clip_gradient ` param has no effect on both SignSGD and Signum, can we just remove this param from `signsgd_update` and `signum_update`?  That would also simply the c++ kernels

Reply:
It has an effect on Signum. Because it will lead to different result whether we use gradient or clipped gradient for calculating momentum.

Reply:
Ah, I see. Thanks for the explanation!